Tests/XCUITests/IntegrationTests.swift

```diff
@@ -189,7 +184,6 @@ class IntegrationTests: BaseTestCase {
     }
 
     func testFxASyncPasswordDesktop () {
```

Comment:
This test is failing for me. It cann't find the secureTextField["Passcode field"].. looks like that has changed? I changed L#200 with `let passcodeInput = springboard.secureTextFields.firstMatch` and then it works... 
But if the test passes for you as it is now, it is fine as it is...

Reply:
That particular line is failing for me once but passing other times. :-/ I can't reproduce the failure reliably.